manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MicrosoftStudios.MinecraftEducationEdition

```

```suggestion
PackageIdentifier: Mojang.MinecraftEducationEdition
```

(Note: You'll have to move it to the correct folder as well).